velox/docs/configs.rst

```diff
@@ -414,6 +414,16 @@ Each query can override the config by setting corresponding query session proper
      - bool
      - true
      - If true, the partition directory will be converted to lowercase when executing a table write operation.
+   * - allow-null-partition-keys
+     - allow_null_partition_keys
+     - bool
+     - true
+     - Determines whether null values for partition keys are allowed or not. if false, it will threw error message
+       like "Partition key must not be null" when writing data with null partition key.
+       **Important Note:**
+       the check for null partition keys should be applied only when partition keys are generated dynamically at
```

Comment:
Null check for partitioning key should be used only when partitions are generated dynamically during query execution. For queries ...

Reply:
good advice, fixed.